src/test/java/teammates/sqlui/webapi/GetFeedbackResponseCommentActionTest.java

```diff
@@ -0,0 +1,328 @@
+package teammates.sqlui.webapi;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.ArrayList;
+
+import org.apache.http.HttpStatus;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.FeedbackParticipantType;
+import teammates.common.datatransfer.InstructorPrivileges;
+import teammates.common.util.Const;
+import teammates.common.util.StringHelper;
+import teammates.storage.sqlentity.Account;
+import teammates.storage.sqlentity.Course;
+import teammates.storage.sqlentity.FeedbackQuestion;
+import teammates.storage.sqlentity.FeedbackResponse;
+import teammates.storage.sqlentity.FeedbackResponseComment;
+import teammates.storage.sqlentity.FeedbackSession;
+import teammates.storage.sqlentity.Instructor;
+import teammates.storage.sqlentity.Student;
+import teammates.ui.output.FeedbackResponseCommentData;
+import teammates.ui.output.MessageOutput;
+import teammates.ui.request.Intent;
+import teammates.ui.webapi.EntityNotFoundException;
+import teammates.ui.webapi.GetFeedbackResponseCommentAction;
+import teammates.ui.webapi.JsonResult;
+
+/**
+ * SUT: {@link GetFeedbackResponseCommentAction}.
+ */
+public class GetFeedbackResponseCommentActionTest extends BaseActionTest<GetFeedbackResponseCommentAction> {
+
+    private Instructor instructorOfCourse1;
+    private Instructor instructorOfCourse2;
+    private FeedbackResponse responseForQ1;
+    private FeedbackResponse responseForQ2;
+    private FeedbackSession feedbackSessionInCourse1;
+    private Student studentInCourse1;
+    private Student studentInCourse2;
+    private FeedbackResponseComment commentForQ1Response1;
+    private FeedbackResponseComment commentForQ2Response1;
+
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.RESPONSE_COMMENT;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return GET;
+    }
+
+    @BeforeMethod
+    void prepareTestData() {
```

Comment:
Then this setup method in the new test case can be renamed as `setUp()` for consistency